### PR TITLE
Add multi container support

### PR DESCRIPTION
**Pull request by vaibhav-kapstan**
- Adds support for updating multiple container image tags

### CODE REVIEW

**Review by harshit-kapstan — APPROVED** [review]

File: entrypoint.sh [hunk 1]
```diff
@@ -27,13 +37,14 @@ EOF
...
-  DEPLOYMENT_ID=$(cat $filePath | jq -r '.deployment_id')
   
-  if [[ -z "$DEPLOYMENT_ID"  || $STATUS_CODE != 2* ]];
+  
+  if [[ $STATUS_CODE != 2* ]];
   then
     echo "Failed to deploy app, err: $(cat $filePath)"
     exit 1
   fi
+  DEPLOYMENT_ID=$(cat $filePath | jq -r '.deployment_id')
```
Comment (vikas-kapstan) [inline entrypoint.sh:47]: why is this moved?

**Review by vikas-kapstan — APPROVED** [review]

File: entrypoint.sh (hunk 1 above)
Comment (vaibhav-kapstan) [reply entrypoint.sh:47]: deployment id is not available if status wasn't 2XX, hence moved it after 2XX check
